yaksh/static/yaksh/js/add_quiz.js

```diff
@@ -9,3 +9,22 @@ function test()
         document.getElementById("submit").innerHTML = "Save";
     }
 }
+
+String.prototype.beginsWith = function (string) {
+    return(this.indexOf(string) === 0);
+};
+
+function usermode(location)
+{
+  var select = document.getElementById("id_prerequisite");
+  var select_text = select.options[select.selectedIndex].text;
+  if (select_text.beginsWith("----")){
+  		window.alert("No prerequisite for this course.\n \
```

Comment:
Not sure if window.alert is the best way to do this, perhaps this can be changed later to use notify.js or some other library.